filter_finalreport.py

```diff
@@ -0,0 +1,47 @@
+#! /usr/bin/env python3
+
+import argparse
+import glob
+
+
+def slice_file(input_file, chromosomes):
+    write_file = open("{}_sliced.txt".format(input_file[:-4]), "w")
+    input_lines = open(input_file, "r")
+    chrom_list = chromosomes.split(",")
+
+    """ Write all lines until Data section """
```

Comment:
I dont full understand why you comment like this (personal preference?)
https://peps.python.org/pep-0008/

Reply:
No particular reason. Changed it!